clients/da-vinci-client/src/main/java/com/linkedin/davinci/blobtransfer/server/P2PFileTransferServerHandler.java

```diff
@@ -201,10 +206,13 @@ private void sendFile(File file, ChannelHandlerContext ctx) throws IOException {
     ChannelFuture sendFileFuture;
     ChannelFuture lastContentFuture;
     long length = raf.length();
+    String fileChecksum = BlobTransferUtils.generateFileChecksum(file.toPath());
+
     HttpResponse response = new DefaultHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.OK);
     response.headers().set(HttpHeaderNames.CONTENT_LENGTH, length);
     response.headers().set(HttpHeaderNames.CONTENT_TYPE, APPLICATION_OCTET_STREAM);
     response.headers().set(HttpHeaderNames.CONTENT_DISPOSITION, "attachment; filename=\"" + file.getName() + "\"");
+    response.headers().set(HttpHeaderNames.CONTENT_MD5, "attachment; checksum=\"" + fileChecksum + "\"");
```

Comment:
Is it necessary to use this value prefix: `attachment; checksum=`? Can we just store the checksum as the full value?